### DISCUSSION TITLE
AI Auto-Tag and Highlighting

### DISCUSSION THREAD

**Discussion by llamapower89** [pfrankov/obsidian-local-gpt#31] [Ideas]
Would it be possible to set-up a an  auto-routine for a paper, imported in MD format, to automatically read and convert keywords and key phrases into tags for my vault? (e.g. The cat ran up the tree. > The #Cat #ran up the #tree.) This would help since tagging is complex and time consuming, so getting an AI to aid me when reading a paper (many of them are 100's of pages long) before I make my notes would be helpful and save hours of work.

**Comment by pfrankov** [comment]
Sorry but this is definitely edge case. You can try to make external AI Agents (ask YouTube) for this task.
